addons/strings/fnc_formatNumber.sqf

```diff
@@ -88,4 +88,4 @@ if (_separateThousands) then { // add localized thousands seperator "1,000"
 
 if (_isNegative) then {_return = "-" + _return;}; // re-add negative sign
 
-_return
+_return
```

Comment:
Am I the one who don't like this "last return" deleting?

Reply:
Not just you, there should be an empty newline.